openshift_scalability/content/centos-stress/Dockerfile

```diff
@@ -14,19 +20,22 @@ RUN yum install -y bc gcc git gnuplot go java-1.8.0-openjdk make \
     yum remove gcc go make patch -y && \
     yum clean all
 
-# Setup jmeter
-RUN mkdir -p /opt/jmeter && \
+### Setup JMeter
+RUN mkdir -p ${APP_ROOT} && \
       curl -Ls https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-3.0.tgz \
-	| tar xz --strip=1 -C /opt/jmeter && \
-        echo "jmeter.save.saveservice.url=true" >> /opt/jmeter/bin/jmeter.properties && \
-        echo "jmeter.save.saveservice.thread_counts=true" >> /opt/jmeter/bin/jmeter.properties && \
-	echo "jmeter.save.saveservice.autoflush=true" >> /opt/jmeter/bin/user.properties && \
-	ln -s /opt/jmeter/bin/jmeter.sh /usr/bin/jmeter && \
+	| tar xz --strip=1 -C ${APP_ROOT} && \
+        echo "jmeter.save.saveservice.url=true" >> ${APP_ROOT}/bin/jmeter.properties && \
+        echo "jmeter.save.saveservice.thread_counts=true" >> ${APP_ROOT}/bin/jmeter.properties && \
```

Comment:
Is this indentation inconsistent for a reason?

Reply:
Will change.